File: javascript/BAEKJOON/BRONZE Ⅲ/13225 Divisors.js
var input = require('fs').readFileSync(__dirname+'/test.txt').toString().trim().split("\n");
//var input = require('fs').readFileSync("/dev/stdin", "utf-8").trim().split("\n");

const C = parseInt(input[0]);
for(let i = 0; i < C; i++){
    const N = parseInt(input[i+1]);

    let result = 0;
    for(let num = 1; num <= Math.sqrt(N); num++) {
        if(N % num === 0){
            result += num !== N / num? 2 : 1;
        }
    }

    console.log(`${N} ${result}`);
}

File: Python/BAEKJOON/BRONZE Ⅲ/13225 Divisors.py
import sys
input = sys.stdin.readline

for _ in range(int(input())):
    N = int(input())

    result = 0
    for num in range(1, int(N ** 0.5) + 1):
        if N % num == 0:
            result += 2 if num != N / num else 1

    print("{} {}".format(N, result))